js/menu.js

```diff
@@ -1,18 +1,31 @@
-$('.show-menu-button').on('click', function() {
+function showMenu() {
 	$('.ui.vertical.sidebar').addClass('overlay visible');
+}
+
+function hideMenu() {
```

Comment:
@npari This was already done. This must have happened due to the resolution of merge conflicts (last commit in this PR). This issue arises while updating the work in progress branch with develop branch.

![selection_005](https://user-images.githubusercontent.com/13470643/29374274-73d19bda-82ce-11e7-9364-1810ad4d9bb1.png)